### ISSUE TITLE
[Feature Request] Improve usability in Shortcut mapper: Remove the Numbers column

### ISSUE THREAD

**Issue by rillig** [notepad-plus-plus/notepad-plus-plus#5609]
Extracted from #2563:

## Remove the numbers column

Currently there is a number column to the left of the Name column. Its purpose is not obvious. It might be a help for remembering the current position when scrolling in the list.

Removing this column provides more space for the remaining columns.

**Comment by sasumner** [comment]
> It might be a help for remembering the current position when scrolling in the list.

Yes.

Also helps in locating conflicts.

Also helps when doing tech support with noobs -- one can say "look for 27" or whatever instead of "look for SCI_SETWRAPINDENTSTYLE" or whatever (I just made those up).

Recommend rejection and closure.

**Comment by rillig** [comment]
There is surely a better way of resolving shortcut conflicts than to tell the user "look in lines 29 and 624, where you will find the conflicting shortcuts". This is incredibly inconvenient.

Therefore I recommend keeping this open and researching for which other bad workarounds the numbers might be used. There should be no legitimate workflow that requires these numbers.

**Comment by sasumner** [comment]
You combined two distinct points from my earlier comments into one; that was not my intent. Let me try again:

"helps in locating conflicts" -- **a number helps me instantly know which way to go to find the conflict, up or down (nothing to do with helping noobs)**.

"helps when doing tech support with noobs -- one can say "look for 27" or whatever instead of "look for SCI_SETWRAPINDENTSTYLE" or whatever" -- **nothing to do with conflicting shortcuts, it's just helpful to give noobs something they understand and can grab onto (a number) rather than a cryptic-to-them monikker**.

Sure, if the numbers weren't already there I would not miss them.  But since they are there they should stay.

**Comment by MetaChuh** [comment]
@rillig 

many thanks for your feature request.

unfortunately we can not remove the number column, as it is necessary for help desk providers, requiring a specific shortcut item number, when users are in non filtered or specifically keyword filtered view.

best regards.
